CliClient/tests/models_Setting.js

```diff
@@ -31,4 +31,19 @@ describe('models_Setting', function() {
 		expect('username' in output).toBe(false);
 	}));
 
+	it('should save values', asyncTest(async () => {
+		await setupDatabase();
+
+		let key = 'sync.5.username';
+		let value = 'testUser';
+		await Setting.setValue(key, value);
+		let savedValue = Setting.value(key);
+		expect(savedValue).toBe(value);
+
+		key = 'sync.5.password';
+		value = 'Test$123';
+		await Setting.setValue(key, value);
+		savedValue = Setting.value(key);
+		expect(savedValue).toBe(value);
```

Comment:
I don't think this tests anything related to this feature, because the test env may or may not have a keychain.

Reply:
I'm having trouble writing a test for this. I am not sure what exactly I should be testing. Can you tell what I should be testing for? Should I be directly retrieving from the system keychain (if present) and test? I need some help here!